src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -2980,16 +2979,19 @@ namespace PackagedTerminalHeatPump {
 			MySizeFlag( PTUnitNum ) = false;
 		}
 
+		RhoAir = StdRhoAir;
```

Comment:
@nealkruis Correct on the standard mass flow rates, no adjustments..  That's the current method for all fans/flows as I understand it.